src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  const getSum = (a) => {
+    count++;
+
+    const limit = count > 3 && count % 2 === 0;
 
+    return (b) => {
+      if (limit) {
+        return 'Bzzz... Error!';
+      }
+
+      return a + b;
```

Comment:
```suggestion
      return limit
        ? 'Bzzz... Error!'
        : a + b
```

Reply:
and if we speak of boolean variables - their names should start with words like `is...`, `should...`, `can...` etc.

Here I would use sth like `shouldBreak`

Reply:
Done! Thanks Nazar!